lib/repo_checks

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Can you update this file with the file from https://github.com/crowbar/crowbar/pull/2081?